File: backend/Spring/exchange/src/main/java/iset/project/exchange/entities/Association.java
package iset.project.exchange.entities;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Document
public class Association {
	@Id
	private String id;
	
	private String nom_association;
	private String matricule;
	private String num_tel;
	private String email;
	private String mot_de_passe;
}

File: backend/ExpressJS/Models/Association.js
const mongoose = require("mongoose");

const associationSchema = new mongoose.Schema({
    nom_association: {
        type: String,
        required: true
    },
    matricule: {
        type: String,
        required: true
    },
    num_tel: {
        type: String,
        required: true
    },
    email: {
        type: String,
        required: true
    },
    mot_de_passe: {
        type: String,
        required: true
    },
});
module.exports = mongoose.model("association", associationSchema);